daemon/feed/FeedCoordinator.h

```diff
@@ -31,6 +31,7 @@
 #include "Observer.h"
 #include "Util.h"
 
+#include <condition_variable>
```

Comment:
For this and other places: please move all new includes into [nzbget.h](https://github.com/nzbget/nzbget/blob/develop/daemon/main/nzbget.h#L225).

This improves compiling times with precompiled headers.

Reply:
done